packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -682,6 +707,114 @@ class Viewport implements IViewport {
+      // It needs to be adjusted by the sum of:
+      // vector between center and initialCenter in image size coordinates
+      // vector between the canvas center and the specific canvasCenter
+      if (initialCenter[0] !== 0.5) {
+        const delta = 0.5 - initialCenter[0];
+        const viewRight = vec3.create();
+        vec3.cross(viewRight, viewPlaneNormal, viewUp);
+        vec3.scaleAndAdd(
+          focalPoint,
+          focalPoint,
+          viewRight,
+          delta * widthWorld * worldSizeError
+        );
+      }
+      if (initialCenter[1] !== 0.5) {
+        const delta = 0.5 - initialCenter[1];
+        const viewUp = activeCamera.getViewUp() as Point3;
+        vec3.scaleAndAdd(
+          focalPoint,
+          focalPoint,
+          viewUp,
+          delta * heightWorld * worldSizeError
+        );
+      }
+      const topLeft = this.canvasToWorld([0, 0]);
+      if (canvasCenter[0] !== 0.5) {
+        const delta = 0.5 - canvasCenter[0];
+        const right = this.canvasToWorld([this.sWidth, 0]);
+        const vector = subtractVec3(right, topLeft);
+        vec3.scaleAndAdd(focalPoint, focalPoint, vector, delta);
+      }
+      if (canvasCenter[1] !== 0.5) {
+        const delta = 0.5 - canvasCenter[1];
+        const bottom = this.canvasToWorld([0, this.sHeight]);
+        const vector = subtractVec3(bottom, topLeft);
+        vec3.scaleAndAdd(focalPoint, focalPoint, vector, delta);
+      }
+      activeCamera.setFocalPoint(focalPoint[0], focalPoint[1], focalPoint[2]);
+      activeCamera.setPosition(
+        focalPoint[0] + distanceVec[0],
+        focalPoint[1] + distanceVec[1],
+        focalPoint[2] + distanceVec[2]
+      );
+    }
+
+    this.panOffset = [0, 0];
+    this.panOffset = this.getCanvasPan();
+  }
+
+  // The initial canvas pan of 0,0 means
+  // that the initialCenter corresponds to the canvasCenter, as relative values
+  public getCanvasPan(): Point2 {
+    const activeCamera = this.getVtkActiveCamera();
+    const focalPoint = <Point3>activeCamera.getFocalPoint();
+    const center = <Point2>[0, 0];
+    const center3 = this.canvasToWorld(center);
+    const delta = this.worldToCanvas(subtractVec3(focalPoint, center3));
+    const result = subtractVec2(delta, this.panOffset);
+    return result;
+  }
+
+  /** Sets the canvas pan value relative to the initial view position of 0,0 */
+  public setCanvasPan(pan: Point2) {
+    const previousCamera = this.getCamera();
+    const { focalPoint, position } = previousCamera;
+    const destDelta = this.getCanvasPan();
+    const delta = subtractVec3(
+      this.canvasToWorld(destDelta),
+      this.canvasToWorld(pan)
+    );
+
+    const newFocal = addVec3(delta, focalPoint);
+    const newPosition = addVec3(delta, position);
+    this.setCamera({
+      ...previousCamera,
+      focalPoint: newFocal,
+      position: newPosition,
+    });
+  }
+
+  /** Returns a current zoom level relative to the base zoom */
```

Comment:
Returns the parallel scale value relative to the original parallel scale.  That is, returns 1 on initial display, and the to magnify the image, values larger than 1 can be set (eg 2 would be twice the original size)